apps/betterangels/src/app/(tabs)/_layout.tsx

```diff
@@ -249,6 +249,7 @@ export default function TabLayout() {
       <MainPlusModal closeModal={closeModal} isModalVisible={isModalVisible} />
       <ConsentModal
         user={user}
+        height={'90%'}
```

Comment:
I would rather do only offset from top or offset + what ui needs. 
90% will look different from device to device. 

const insets = useSafeAreaInsets();
const topOffset = insets.top;